Evaluation criteria improvement (#4)
OriginFinancial/origin-backend-take-home-assignment: README.md

```

## Criteria
You may use any language and framework provided that you build a solid system with an emphasis on code quality, simplicity, readability, maintainability, and reliability, particularly regarding architecture and testing. We'd prefer it if you used Python, but it's just that – a preference.

Be aware that Origin will mainly take into consideration the following evaluation criteria:
* How clean and organized your code is;
* If you implemented the business rules correctly;
* How good your automated tests are (qualitative over quantitative).

Other important notes:

> * Develop an extensible recommendation engine, it will be essential to the take-home interview

Reviewer: I'd drop the spoiler. I would let just "Develop an extensible recommendation engine."

Reviewer: About the spoiler, I think you mean `tech on-site` instead of `take-home interview`. Anyway, i'd drop it!

> * Add to README file (1) what were the main technical decisions you made; (2) relevant comments about your project; (3) instructions to run the code 

Reviewer: As we are enumerating, I would bring to the front `(1) instructions to run the code`, since it is our basic criteria.

> * You must use English in your code and also in your docs, including the README file

Reviewer: I think the README file here is redundant, I could be just `You must use English in your code and also in your docs.`

Author: Agreed

```

## Criteria
You may use any language and framework provided that you build a solid system with an emphasis on code quality, simplicity, readability, maintainability, and reliability, particularly regarding architecture and testing. We'd prefer it if you used Python, but it's just that – a preference.

Be aware that Origin will mainly take into consideration the following evaluation criteria:
* How clean and organized your code is;
* If you implemented the business rules correctly;
* How good your automated tests are (qualitative over quantitative).

Other important notes:

> * Develop a flexible recommendation engine, it will be essential to the take-home interview

Reviewer 2: ```suggestion
* Develop an extensible recommendation engine, it will be essential to the take-home interview
```

It's just a word, but I think the difference is important:
* Flexible is something that is aimed to be CHANGED
* Extensible is the Open-Closed Principle, something that we strongly care about during the Take Home Review. "How simple is to add a new rule or line of insurance? Does it take serious changes to the core implementation?"
